partitioning/gpt/gpt.go

```diff
@@ -600,6 +605,52 @@ func (t *Table) writePMBR() error {
 }
 
 func (t *Table) syncKernel() error {
+	if t.newTable {
+		return t.syncKernelComplete()
+	}
+
+	return t.syncKernelIncremental()
+}
+
+// syncKernelComplete synchronizes the kernel partition table with the current table by overwriting the whole table.
+//
+// It is incompatible with mounted partitions.
```

Comment:
Wasn't there a syscall to just tell the kernel to re-read the partition table? Also we must probably check this is not running on a disk which has a mounted partition and return an error if so

Reply:
And by that syscall I meant BLKRRPART to be clear